scripts/dependencies.props

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <MSTestVersion>2.1.1</MSTestVersion>
+    <MSTestVersion>2.2.8</MSTestVersion>
     <NETTestSdkVersion>16.7.1</NETTestSdkVersion>
     <MoqVersion>4.9.0</MoqVersion>
     <CoverletVersion>2.7.0</CoverletVersion>
 
     <!-- Test Assets use the minimum supported versions -->
     <NETTestSdkMinimumVersion>15.0.0</NETTestSdkMinimumVersion>
-    <XunitVersion>2.3.1</XunitVersion>
-    <XunitTestAdapterVersion>2.3.1</XunitTestAdapterVersion>
+    <XunitVersion>2.4.1</XunitVersion>
+    <XunitTestAdapterVersion>2.4.3</XunitTestAdapterVersion>
```

Comment:
I had bumped up these versions but realize you tagged them as the lowest supported. I could revert this.

Reply:
Rolled this back